### DISCUSSION TITLE
Radarr: Errors when starting with scripts_init.bash

### DISCUSSION THREAD

**Discussion by DataLoreQ** [RandomNinjaAtk/arr-scripts#179] [Q&A] [answered]
When I do not have scripts_init.bash in custom-cont-init.d, Radarr works normally.  However, when I have scripts_init.bash in the directory there are errors in the log, otherwise Radarr works normally.

```
[custom-init] Files found, executing
[custom-init] scripts_init.bash: executing...
/custom-cont-init.d/scripts_init.bash: line 2: $'bash\r': command not found
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0  4209    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (23) Failure writing output to destination
/custom-cont-init.d/scripts_init.bash: line 3: $'exit\r': command not found
[custom-init] scripts_init.bash: exited 127
```
I did follow this:
```
Add 2 volumes to your container /custom-services.d and /custom-cont-init.d (do not map to the same local folder...) Docker Run Example: -v /path/to/preferred/local/folder-01:/custom-services.d -v /path/to/preferred/local/folder-02:/custom-cont-init.d
Download the [script_init.bash](https://github.com/RandomNinjaAtk/arr-scripts/blob/main/radarr/scripts_init.bash) ([Download Link](https://raw.githubusercontent.com/RandomNinjaAtk/arr-scripts/main/radarr/scripts_init.bash)) and place it into the following folder: /custom-cont-init.d
Start your container and wait for the application to load
```

Any assistance would be appreciated, as I'm sure I am doing something wrong.

Thank you,
DataLoreQ

**Comment by RandomNinjaAtk** [comment, marked as answer]
You either modified the script or manually created it and it has the wrong line endings. Most likely you have Windows Line endings and not Linux/Unix compatible line endings which is required because the container is a linux based OS.

If you manually created the script by copying the code, use a text editor (vscode, notepad++, etc...) that can change the line endings and resave the file.

OR 

Download the raw file, save it and move it to the correct location without modifying it, like the instructions say... (https://raw.githubusercontent.com/RandomNinjaAtk/arr-scripts/main/radarr/scripts_init.bash)
